internal/pkg/cli/env_package_test.go

```diff
@@ -280,6 +281,7 @@ func TestPackageEnvOpts_Execute(t *testing.T) {
 					},
 					fs:     fs,
 					envCfg: &config.Environment{Name: "test"},
+					appCfg: &config.Application{},
```

Comment:
nit: perhaps we could augment the unit test by having
```
appCfg: &config.Application{
  permissionsBoundary: "mockPermissionsBoundary",
},
```

and then have in line 262
```
deployer.EXPECT().GenerateCloudFormationTemplate(gomock.Any()).Return(&deploy.GenerateCloudFormationTemplateOutput{
					Template:   "template",
					Parameters: "parameters",
					PermissionsBoundary: "mockPermissionsBoundary",
				}, nil)
```

Reply:
done.